solutions/swb-reference/src/services/databaseService.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DynamoDBService, JSONValue, MetadataService } from '@aws/workbench-core-base';
+
+export interface DatabaseServicePlugin {
```

Comment:
It would be lovely to have doc strings for this file so people can know how to use it!

Reply:
discussed on slack: these will come after we get our stories done for release.